clients/privacy-center/common/hooks/useI18n.ts

```diff
@@ -0,0 +1,51 @@
+import { useCallback, useContext } from "react";
+import {
+  ExperienceConfig,
+  PrivacyNotice,
+  selectBestExperienceConfigTranslation,
+  selectBestNoticeTranslation,
+} from "fides-js";
+import { I18nContext } from "../i18nContext";
+
+const useI18n = () => {
```

Comment:
Added hook for quick access to i18n instance and useful functions that already have the necessary context.